src/libcore/ptr/const_ptr.rs

```diff
@@ -699,7 +707,8 @@ impl<T: ?Sized> *const T {
     where
         T: Sized,
     {
-        read_unaligned(self)
+        // SAFETY: the caller must uphold the safety contract for `read_unaligned`.
```

Comment:
This is an example of a comment that doesn't add a lot of information, but I don't know that I would change it. =)

Reply:
I added this kinds of comments mostly in functions where the call is actually delegated to another function, without any additional elements coming from the content of this function. Like:
```rust
unsafe fn do_x() {
    unsafe { actually_do_x() }
}
```

In most of these cases, either the safety conditions are written in the doc comments for `do_x` because it is a "public interface" function, or there are no comments anywhere because it's an entirely internal function and that makes it really difficult to reason about safety contracts and I was unable to write more detailed comments.